opencascade[core]:x64-linux build failure

**Host Environment**
- OS: Fedora 34
- Compiler: gcc (GCC) 11.2.1 20210728 (Red Hat 11.2.1-1)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install opencascade`

**Failure logs** 
Computing installation plan...
The following packages will be built and installed:
    opencascade[core]:x64-linux -> 7.5.0#1
Detecting compiler hash for triplet x64-linux...
Could not locate cached archive: /home/chris/.cache/vcpkg/archives/80/8092b814e8857b6a7bbc0152c0b22428e4d731e6ea4a70757fa1f4abc667a1d4.zip
Starting package 1/1: opencascade:x64-linux
Building package opencascade[core]:x64-linux...
CMake Error at scripts/cmake/vcpkg_fail_port_install.cmake:96 (message):
  Target 'Linux' not supported by opencascade!

Call Stack (most recent call first):
  ports/opencascade/portfile.cmake:1 (vcpkg_fail_port_install)
  scripts/ports.cmake:141 (include)


Error: Building package opencascade:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: opencascade:x64-linux
  Vcpkg version: 2021-08-03-unknownhash

**Additional context**

From, https://vcpkg.io/en/packages.html

opencascade | Version: 7.5.0
Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.
Compatibility:
✓ arm-uwp
✓ arm64-windows
✓ x64-linux
✓ x64-osx
✓ x64-uwp
✓ x64-windows
✓ x64-windows-static
✓ x86-windows

--- a/ports/opencascade/portfile.cmake
+++ b/ports/opencascade/portfile.cmake
@@ -69,8 +69,10 @@ foreach(file_name IN LISTS files)
 endforeach()
 
 # Remove libd to lib, libd just has cmake files we dont want too
+if( WIN32 )
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/lib")
 file(RENAME "${CURRENT_PACKAGES_DIR}/debug/libd" "${CURRENT_PACKAGES_DIR}/debug/lib")
+endif()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")

--- a/ports/opencascade/vcpkg.json
+++ b/ports/opencascade/vcpkg.json
@@ -3,7 +3,7 @@
   "version": "7.6.2",
   "description": "Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.",
   "homepage": "https://github.com/Open-Cascade-SAS/OCCT",
-  "license": "LGPL-2.1",
+  "license": "OCCT-exception-1.0",
   "supports": "!(uwp | osx | arm)",
   "dependencies": [
     "fontconfig",

--- a/ports/opencascade/vcpkg.json
+++ b/ports/opencascade/vcpkg.json
@@ -3,7 +3,7 @@
   "version": "7.6.2",
   "description": "Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.",
   "homepage": "https://github.com/Open-Cascade-SAS/OCCT",
-  "license": "OCCT-exception-1.0",
+  "license": "LGPL-2.1",
   "supports": "!(uwp | osx | arm)",
   "dependencies": [
     "fontconfig",

--- a/ports/opencascade/vcpkg.json
+++ b/ports/opencascade/vcpkg.json
@@ -6,6 +6,7 @@
   "license": "LGPL-2.1",
   "supports": "!(uwp | osx | arm)",
   "dependencies": [
+    "fontconfig",
     "freetype",
     {
       "name": "vcpkg-cmake",

--- a/ports/opencascade/vcpkg.json
+++ b/ports/opencascade/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "opencascade",
   "version": "7.6.2",
+  "port-version": 1,
   "description": "Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.",
   "homepage": "https://github.com/Open-Cascade-SAS/OCCT",
   "license": "LGPL-2.1",

--- a/ports/opencascade/vcpkg.json
+++ b/ports/opencascade/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "Open CASCADE Technology (OCCT) is an open-source software development platform for 3D CAD, CAM, CAE.",
   "homepage": "https://github.com/Open-Cascade-SAS/OCCT",
   "license": "LGPL-2.1",
-  "supports": "!(uwp | osx | linux | arm)",
+  "supports": "!(uwp | osx | arm)",
   "dependencies": [
     "freetype",
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5110,7 +5110,7 @@
     },
     "opencascade": {
       "baseline": "7.6.2",
-      "port-version": 0
+      "port-version": 1
     },
     "opencc": {
       "baseline": "2020-04-26",

--- a/versions/o-/opencascade.json
+++ b/versions/o-/opencascade.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "6dd7a80ecf7e8183ebfd1e25fc64351e47f9eed0",
+      "git-tree": "42989455308a507d1e7cd2a8509ffd1b1e7ec885",
       "version": "7.6.2",
       "port-version": 0
     },

--- a/versions/o-/opencascade.json
+++ b/versions/o-/opencascade.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "137097588b2328c3d7f66c4f0e46e6e4fe1559cf",
+      "version": "7.6.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "42989455308a507d1e7cd2a8509ffd1b1e7ec885",
       "version": "7.6.2",
       "port-version": 0

--- a/versions/o-/opencascade.json
+++ b/versions/o-/opencascade.json
@@ -6,7 +6,7 @@
       "port-version": 1
     },
     {
-      "git-tree": "42989455308a507d1e7cd2a8509ffd1b1e7ec885",
+      "git-tree": "6dd7a80ecf7e8183ebfd1e25fc64351e47f9eed0",
       "version": "7.6.2",
       "port-version": 0
     },

